source/_integrations/reolink.markdown

```
@@ -476,6 +476,101 @@
+- [Android or iOS companion app](https://companion.home-assistant.io/docs/getting_started#setting-up)
+- [Remote access to Home Assistant](https://www.home-assistant.io/docs/configuration/remote/). Although you can receive text notifications without remote access, to see the camera image in the notification (rich notification), the phone needs to be able to reach Home Assistant. The rich notification will always work, even without remote access, when the phone is on the same network as Home Assistant.
+
+1. In order to receive such a rich notification we are going to make a automation in Home Assistant. In Home Assistant go to **Settings** > **Automations & scenes** > **+ Create automation** > **Create new automation**
+
+![afbeelding](https://github.com/user-attachments/assets/163c45a5-1027-4ff9-b841-977b9392a026)
+![afbeelding](https://github.com/user-attachments/assets/06a05369-73c6-4994-bdbf-f8b3dc843d5d)
+![afbeelding](https://github.com/user-attachments/assets/8a12d091-d5fa-4a2c-abc2-d01f4fde1ff2)
+
+![afbeelding](https://github.com/user-attachments/assets/b374098e-b4bb-4473-b917-635b3d020b03)
+
+2. Under **When** select: **+ Add trigger** > **Entity** > **State**.
```

Markdownlint (0.35.0)

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)